giraffe/src/utils/fromFlux.ts

```diff
@@ -248,6 +248,10 @@ const parseValue = (value: string | undefined, columnType: ColumnType): any => {
     return undefined
   }
 
+  if (value === '+Inf' || value === '-Inf') {
+    return value
+  }
+
```

Comment:
What if in the future Infinity is expressed differently, for example: ∞

This will lead to the same problem again.

I made a suggestion for this fix in this pull request: https://github.com/influxdata/giraffe/pull/505